sanityv3/src/previews/PreviewWrapper.tsx

```diff
@@ -54,3 +54,4 @@ export const PreviewWrapper = ({ src, children, shareable = true }: Props) => {
     </StyledWrapper>
   )
 }
+export default PreviewWrapper
```

Comment:
why was this needed? 🤔

Reply:
to be honest, i do not know, reversed this